src/ProcessMaker/Nayra/Contracts/Repositories/DataStoreCollectionRepositoryInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ProcessMaker\Nayra\Contracts\Repositories;
+
+
+interface DataStoreCollectionRepositoryInterface extends RepositoryInterface
```

Comment:
A repository for the collection is not required, please use DataStoreRepository and DataStoreCollection